packages/app/src/components/DBRowDataPanel.tsx

```diff
@@ -112,16 +113,40 @@ export function useRowData({
       enabled: rowId != null,
     },
   );
+
+  // Normalize resource and event attributes to always use flat keys for both JSON and Map columns
```

Comment:
The fix makes sense to me. We normalize the attributes structure upstream so that the downstream chart component can render properly.